app/Http/Controllers/Users/UsersController.php

```diff
@@ -291,6 +295,9 @@ public function update(SaveUserRequest $request, $id = null)
 
         $user->permissions =  json_encode($permissions_array);
 
+        app('App\Http\Requests\ImageUploadRequest')->handleImages($user, '', 'avatar', 'avatars');
```

Comment:
@inietov This is how we invoke the second form request from within the controller